src/v2/components/ConnectionSelection/components/ConnectCTA/queries/canConnect.ts

```diff
@@ -0,0 +1,33 @@
+import gql from 'graphql-tag'
+
+export const canConnectChannelQuery = gql`
+  query CanConnectChannelQuery($id: ID!) {
+    connectable: channel(id: $id) {
+      __typename
+      id
+      can {
+        connect
+      }
+    }
+  }
+`
+export const canConnectBlockQuery = gql`
+  query CanConnectBlockQuery($id: ID!) {
+    connectable: blokk(id: $id) {
+      __typename
+      ... on Model {
+        id
+      }
+      ... on Block {
+        can {
+          connect
+        }
+      }
+      ... on Channel {
+        can {
+          connect
+        }
```

Comment:
These will just return `can.connect` => `true` for now until we deploy the ability changes. So this is safe for `master`.